gnomad/utils/constraint.py

```diff
@@ -0,0 +1,162 @@
+# noqa: D100
```

Comment:
I'm leaning keeping the current name: it would be awesome if we could drop all constraint (LoF, RMC) related functions into this script in the future. Also, `annotate_with_mu` and `count_variants` are fairly generic functions (they're mostly used to build the models used inthe LoF and RMC pipelines, so not specific to just LoF)

Reply:
Works for me